dts/bindings/gpio/gpio-leds.yaml

```diff
@@ -15,3 +23,7 @@ child-binding:
           required: true
           type: string
           description: Human readable string describing the device (used by Zephyr for API name)
+       id:
```

Comment:
Looking at https://www.kernel.org/doc/Documentation/devicetree/bindings/leds/leds-gpio.txt the property **id** could be named **function** instead. **functions** seems to have the same intention.